1001-2000/LIT1297Dersan.xml

```diff
@@ -39,64 +39,139 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
+               It consists of a <ref type="work" corresp="LIT7214HomilyJCRuf">Homily by John Chrysostom</ref>, two collections of miracles 
+               (the unity of the two collections is still to be investigated), two additional texts
+               (respectively entitled <ref type="work" corresp="LIT7215HistoryRuf">History of the consecration of the church of Rufāʾel on an island near Alexandria</ref> 
+               and <ref type="work" corresp="LIT7216HistoryJCRuf">History of how John Chrysostom visited the church of Rufāʾel on an island</ref>) and a 
+               <foreign xml:lang="gez">salām</foreign>-hymn entitled <ref type="work" corresp="LIT6633SalRufMastafessehe">Salutation to Raphael, who is called 'the Rejoicer'</ref>.
+            </p>
+         </abstract>
       </profileDesc>
       <revisionDesc>
          <change who="PL" when="2016-03-21"> Created file from google
             spreadsheet </change>
          <change who="ES" when="2016-02-09">CREATED: text record</change>
          <change who="MKr" when="2020-12-04">Updated transliteration, added summary, relations and bibliography, textparts.</change>
          <change who="MKr" when="2022-11-30">Added reference to work records</change>
+         <change who="MV" when="2024-12-20">Abstract, revised textparts</change>
       </revisionDesc>
    </teiHeader>
    <text xml:base="https://betamasaheft.eu/">
       <body>
-        <div type="bibliography">
-          <listBibl type="secondary">
-          <bibl>
-             <ptr target="bm:Lusini2005EAEDersanaRufael"/>
-          </bibl>
-          </listBibl>
-          <listRelation>
-             <relation name="saws:isAttributedToAuthor" active="LIT1297Dersan" passive="PRS5720JohnChr"/>
-             <relation name="ecrm:P129_is_about" active="LIT1297Dersan" passive="PRS8083Raphael"/>
-          </listRelation>
-        </div>
- <div type="edition"><!-- The incipits below are from https://eap.bl.uk/archive-file/EAP432-1-6 -->
-   <note>The following division into textparts follows <ref type="mss" corresp="BLorient619"/></note>
-      <div type="textpart" subtype="chapter" xml:lang="gez" xml:id="HomilyJC">
-        <div type="textpart" subtype="incipit">
-           <ab>ድርሳን፡ ዘይቤ፡ አብ፡ ብፁዕ፡ አባ፡ ዮሐንስ፡ አፈ፡ ወርቅ፡ ሊቀ፡ ጳጳሳት፡ ዘሀገረ፡ ዓባይ፡ ቍስጥንጥንያ፡
-             ዘይትነበብ፡ በበዓለ፡ ቅዱስ፡ ሩፋኤል፡ ሊቀ፡ መላእክት፡</ab>
+         <div type="bibliography">
+            <listBibl type="secondary">
+               <bibl>
+                  <ptr target="bm:Lusini2005EAEDersanaRufael"/>
+               </bibl>
+            </listBibl>
+            <listRelation>            
+               <relation name="ecrm:P129_is_about" active="LIT1297Dersan" passive="PRS8083Raphael"/>
+            </listRelation>
+            <div type="edition">       
+               <div type="textpart" subtype="chapter" xml:lang="gez" xml:id="LIT7214HomilyJCRuf">
+                  <label>Homily by John Chrysostom on the archangel Rufāʾel</label>
```

Comment:
I think, it is better decide on Raphael or Rufāʾel and use the same name throughout the record.